lib/cylc/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 import os
 import sys
-from parsec import LOG
+from cylc.parsec import LOG
```

Comment:
My horror here. You should now move the logic that defines `LOG` from `lib/cylc/parsec/__init__.py` to this module.

Reply:
Done, logs appear to be working as before.